src/table/dtable.jl

```diff
@@ -245,7 +245,8 @@ show(io::IO, d::DTable) = show(io, MIME"text/plain"(), d)
 function show(io::IO, ::MIME"text/plain", d::DTable)
     tabletype = d.tabletype === nothing ? "unknown (use `tabletype!(::DTable)`)" : d.tabletype
     println(io, "DTable with $(nchunks(d)) partitions")
-    print(io, "Tabletype: $tabletype")
+    println(io, "Tabletype: $tabletype")
+    pretty_table(io, fetch(d))
```

Comment:
One concern I have is that `fetch(d)` might be really expensive (or impossible, it might be bigger than available RAM). We would want to instead have `pretty_table` only render the relevant partitions (usually the first few and last few partitions), since it truncates most of the inner elements for long tables.

@ronisbr what methods would we need to enable this?

Reply:
Hum, this one is a tough question :)

Does `d` here follow Tables.jl API? If so, I think it should just work. PrettyTables basically check for the current size of the terminal and only renders the amount of rows and columns necessary to fill the screen (with some margins especially in the rows).

Reply:
```suggestion
    pretty_table(io, d)
```

Indeed, this seems to work!